src/components/Atoms/Input/index.jsx

```diff
@@ -30,7 +30,7 @@ export const Input = ({ onEditComplete, defaultValue = "" }) => {
 };
 
 
-const StyledInput = styled.input.attrs((props) => ({ ...props, size: "1" }))`
+const StyledInput = styled.input.attrs((props) => ({ ...props }))`
```

Comment:
いや、attrs自体が不要なコードです。

```suggestion
const StyledInput = styled.input`
```